appveyor.yml

```diff
@@ -1,7 +1,7 @@
 init:
   - git config --global core.autocrlf true
 build_script:
-  - build.cmd --quiet --parallel verify
```

Comment:
Is it that we don't want it parallel anymore or does that idea just not exist in the new world?